### PR TITLE
added myNotes Project

### PR DESCRIPTION
**Pull request by nitin-787**

### CODE REVIEW

File: Flutter Open Source Apps/Advanced/readme.md [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+### Did I say the hacktoberfest, yes you heard me right it is participating the hacktoberfest 2022
+### To support the Hacktoberfest event, the hacktoberfest label will be added to every pull request made on 1 to 31 October 2022
+### Let's prepare yourself!
+
+## Why do i need to work this?
+### If you reach till here then I am pretty sure you have this question is in your mind why should I work on this project well, if you are flutter developer then this is perfect project for you, you will get to learn alot things about flutter like Bloc, state management, No internet activity, shimmer loading animation, flutter responsiveness, alot of different types of pakages and what not some some additional things like Firebase, github CI/CD pipeline, google auth.
+
+## can I contribute?
+### Yes bcuz why not, you can contribute through code and non code, one thing I can make does't matter if you're a naive or a master
+### Repo link : https://github.com/nitin-787/mynotes
```
Comment (HemantPra389) [inline Flutter Open Source Apps/Advanced/readme.md:19]: The Repo link should be:-https://github.com/HemantPra389/Flutter-Open-Source-Apps

Comment (nitin-787) [reply Flutter Open Source Apps/Advanced/readme.md:19]: Can elaborate it more?

**Comment by HemantPra389** [comment]
If you really want to contribute with no code and wanna add readme files. Then try to enhance the quality of every readme possible. As per hacktoberfest2022 guidelines the PR must worth the repository.🤗
Happy Hacktoberfest2022 !! 🥳

File: Flutter Open Source Apps/Advanced/readme.md (hunk 1 above)
Comment (nitin-787) [reply Flutter Open Source Apps/Advanced/readme.md:19]: > Can you elaborate it more?

> If you really want to contribute with no code and wanna add readme files. Then try to enhance the quality of every readme possible. As per hacktoberfest2022 guidelines the PR must worth the repository.🤗 Happy Hacktoberfest2022 !! 🥳



> If you really want to contribute with no code and wanna add readme files. Then try to enhance the quality of every readme possible. As per hacktoberfest2022 guidelines the PR must worth the repository.🤗 Happy Hacktoberfest2022 !! 🥳

Hey Sorry I if miss understood but I owner of this repo https://github.com/nitin-787/Flutter-Open-Source-Apps/tree/nitin/Flutter%20Open%20Source%20Apps/Advanced#readme and I thought this repo that contain different type of flutter projects and I just added mine

**Comment by nitin-787** [comment]
> If you really want to contribute with no code and wanna add readme files. Then try to enhance the quality of every readme possible. As per hacktoberfest2022 guidelines the PR must worth the repository.🤗 Happy Hacktoberfest2022 !! 🥳

Hey Sorry I if miss understood but I owner of this repohttps://github.com/nitin-787/mynotes and I thought this repo that contain different type of flutter projects and I just added mine

File: Flutter Open Source Apps/Advanced/readme.md (hunk 1 above)
Comment (nitin-787) [reply Flutter Open Source Apps/Advanced/readme.md:19]: Do I need to add my whole project here?

Comment (HemantPra389) [reply Flutter Open Source Apps/Advanced/readme.md:19]: Yes so that people can find opensource flutter apps at one go

**Pull request closed without merging**